scripts/release.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
All of our release scripts are called `bd_to_prod.sh`

Reply:
Efi is right, can you rename it?